src/test/java/com/twilio/TwilioTest.java

```diff
@@ -135,4 +137,97 @@ public void testValidateSslCertificateSuccess() {
 
         Twilio.validateSslCertificate();
     }
+
+    @Test
+    public void testEdgeNoRegion() {
+        new Expectations() {{
+            final Request request = new Request(HttpMethod.GET, "https://api.edge.us1.twilio.com/test");
+            networkHttpClient.makeRequest(request);
+            times = 1;
+            result = new Response("", 200);
+        }};
+
+        Twilio.init("accountSid", "authToken");
+        Twilio.setEdge("edge");
+        Request request = new Request(HttpMethod.GET, "api", "/test", null, Twilio.getEdge());
+        //Request request = new Request(HttpMethod.GET, "https://api.edge.us1.twilio.com/test");
+        Response result = networkHttpClient.makeRequest(request);
```

Comment:
Why are you use `networkHttpClient` here instead of `TwilioRestClient client = Twilio.getRestClient();` like in [this regional test](https://github.com/twilio/twilio-java/pull/525/files#diff-019d944bb76bdd4be40cc700e5217442R159)?

Reply:
Reverted back, no luck.